src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,95 @@
+import React, { Children, Fragment, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import TabContent from './TabContent';
+import TabItem from './TabItem';
+import TabMenu from './TabMenu';
+
+class Tabs extends PureComponent {
+    constructor(props) {
+        super(props);
+
+        const { activeTabId } = props;
+
+        this.state = {
+            activeTabId,
+            derivedProp: activeTabId // eslint-disable-line react/no-unused-state
+        };
+    }
+
+    static getDerivedStateFromProps(props, state) {
+        if (props.activeTabId === state.derivedProp) {
+            return null;
+        }
+        const { activeTabId } = props;
+        return {
+            activeTabId,
+            derivedProp: activeTabId
+        };
+    }
+
+    handleTabChange(e, tabId) {
```

Comment:
As far as I can tell `e` is commonly used for events (besides catch-statements) ([evidence](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onclick#JavaScript), [more evidence](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events#Event_objects)) and `event` is a global name in IE. What would you suggest otherwise?

Reply:
Updated